websites/#/アイドルマスターシャイニーカラーズ/presence.ts

```diff
@@ -0,0 +1,134 @@
+const presence = new Presence({
+  clientId: "1103931016525127792",
+}),
+  browsingTimestamp = Math.floor(Date.now() / 1000);
+
+presence.on("UpdateData", async () => {
+  const presenceData: PresenceData = {
+    largeImageKey: "https://i.imgur.com/ue8EatG.jpg",
+  };
+
+  const { pathname } = window.location;
+
+  if (pathname === "/home") {
```

Comment:
Most of the paths in this presence are static. Put the static paths in a structure like so:
https://github.com/PreMiD/Presences/blob/236523980322a4053a45c691a1739eb50e2aea48/websites/G/Geotastic/presence.ts#L12-L42